pkg/machine/qemu/machine.go

```diff
@@ -1289,6 +1330,14 @@ func (v *MachineVM) setupAPIForwarding(cmd []string) ([]string, string, apiForwa
 	cmd = append(cmd, []string{"-forward-user", forwardUser}...)
 	cmd = append(cmd, []string{"-forward-identity", v.IdentityPath}...)
 
+	// Workaround for Podman Desktop compatibility
+	if runtime.GOOS == "windows" {
```

Comment:
Workaround to make Podman Desktop happy. Will not be needed when https://github.com/containers/podman/issues/16860 is implemented and Podman Desktop will be updated to use the CLI instead of building the address internally.